lib/cocina_display/concerns/contributors.rb

```diff
@@ -84,7 +84,9 @@ def contributors_by_role(with_date: false)
       # Contributors with no role are grouped under a default heading.
       # @return [Array<DisplayData>]
       def contributor_display_data
-        contributors_by_role.map do |role, contributors|
+        contributors_by_role.filter_map do |role, contributors|
+          next if role == "publisher"
+
```

Comment:
I noticed that PURL does:

```suggestion
        contributors_by_role.except("publisher").filter_map do |role, contributors|
```

Which seems a little cleaner. We can also update in PURL once this is merged.

Reply:
See https://github.com/sul-dlss/purl/blob/39b72e8ab29b5ee484bb61e1e12e5295b0f380c0/app/components/contributors_component.rb#L17-L21